vpr/src/route/check_rr_graph.cpp

```diff
@@ -534,11 +502,11 @@ static void check_unbuffered_edges(int from_node) {
         }
 
         if (trans_matched == false) {
-            VTR_LOG_WARN(
```

Comment:
You reverted this, which is not relevant to the non-consecutive PTC's.

Reply:
Ah right, didn't notice that. I suppose that now this PR is almost done. I will handle all the spots where the `ptc` are used for chanx/y, check against all_xc7 and ice40 and then open a PR upstream.